nbdev/clean.py

```diff
@@ -38,7 +38,7 @@ def nbdev_trust(
     path = fname if fname.is_dir() else fname.parent
     check_fname = path/".last_checked"
     last_checked = os.path.getmtime(check_fname) if check_fname.exists() else None
-    nbs = globtastic(fname, file_glob='*.ipynb', skip_folder_re='^[_.]') if fname.is_dir() else [fname]
+    nbs = globtastic(fname, file_re=r'.*\.ipynb$|.*\.qmd$', skip_folder_re='^[_.]') if fname.is_dir() else [fname]
```

Comment:
We could simplify this in the multiple places it occurs, eg:
```suggestion
    nbs = globtastic(fname, file_re=r'.*\.(ipynb|qmd)$', skip_folder_re='^[_.]') if fname.is_dir() else [fname]
```
Although it feels like a glob would be nicer. What if we added a `file_exts` param to globtastic which took a comma separated list of file extensions? Then we could just have `file_exts='ipynb,qmd'`.